lib/config.js

```diff
@@ -48,7 +48,9 @@ export default class Config {
         baseDir: Joi.string(),
         ui: [Joi.bool(), Joi.object()],
         files: [Joi.string(), Joi.array()],
-        watchOptions: Joi.object(),
+        watchOptions: Joi.object().default().keys({
+          ignored: Joi.array().default(['node_modules'])
```

Comment:
Wont this allow the user to overwrite? I feel like we always want to disable `node_modules` from being watched...